stdlib/flux_test.go

```diff
@@ -149,8 +164,14 @@ func doTestInspect(t testing.TB, c flux.Compiler) {
 			t.Error(err)
 		}
 	}
+	r.Done()
```

Comment:
There is a `defer r.Done()` above?

Reply:
This ensures that the query is done before we take the statistics which is needed to ensure that the statistics are computed. The `defer` is done above to ensure that this always happens even on error conditions. It is safe to call `Done()` multiple times.